app/App.jsx

```diff
@@ -0,0 +1,11 @@
+import React, {Component} from 'react'
+import {render} from 'react-dom'
+
+export default class App extends Component {
```

Comment:
Please ensure that all new/modified components have corresponding tests.
Use [Jest](https://github.com/facebook/jest) for writing tests.

Reply:
Okay, I used Jest and enzyme as documented 
https://facebook.github.io/jest/docs/en/tutorial-react.html